### DISCUSSION TITLE
[1207650] Suzerain | 本体

### DISCUSSION THREAD

**Discussion by harrylisen** [pjy612/SteamManifestCache#450] [Share]
其实我不知道sai是啥，我是意外找到的这个项目
`appid:1207650`
keys:
```
1207651;e962f7f3acaa1caae21e3a4b34d2c88979477e1e5c6343007e2b688814031f1b
```

[清单.zip](https://github.com/pjy612/SteamManifestCache/files/14129032/default.zip)

**Comment by pjy612** [comment]
感谢你的分享
* 但是 密钥样式不对 #26

**Reply by harrylisen** [reply]
应该改好了。
